src/vaadin-grid-drag-and-drop-mixin.html

```diff
@@ -247,6 +253,37 @@
         }
       }
 
+      __dndAutoScroll(clientY) {
+        if (this.__dndAutoScrolling) {
+          return true;
+        }
+
+        const headerBotton = this.$.header.getBoundingClientRect().bottom;
+        const footerTop = this.$.footer.getBoundingClientRect().top;
+        const topDiff = headerBotton - clientY + 50;
```

Comment:
Should we use some property for this instead of just `50`, probably even private? Not a blocker.

Reply:
Done.